app/views/oregon_digital/explore_collections/_list_showcase.html.erb

```diff
@@ -13,8 +13,19 @@
   </thead>
   <tbody>
     <% collections.each_with_index do |collection, i| %>
+      <% institution = collection.institution.first == "http://id.loc.gov/authorities/names/n80017721" ? "OSU" : "UO" %>
+      <% institution = "" if collection.institution.empty? %>
+      <% if Collection.find(collection.id).insitution_restricted?(current_ability) %>
```

Comment:
Maybe save the restricted status so we don't have to do 2x Fedora lookups (even if they're cached)